### PR TITLE
Disclose relationship with TideLift

### PR DESCRIPTION
**Pull request by ppkarwasz**
This is a draft of the changes to the Support page to disclose the relationship of some PMCs with TideLift.

Personally I don't mind disclosing the names of the people involved, if the other PMCs prefer it this way.

### CODE REVIEW

File: src/site/markdown/support.md [hunk 1]
```diff
@@ -49,3 +49,16 @@ projects, we cannot recommend any particular individual for any specific issue.
 * [Ralph Goers](https://github.com/rgoers)
 * [Volkan Yazıcı](https://github.com/vy)
 * [Carter Kozak](https://github.com/carterkozak)
+
+## Third-party commercial support
+
+While neither the Apache Software Foundation nor the Apache Logging Services project provide any commercial support for
+the Log4j products individual committers may collaborate with services that provide such support.
```
Comment (vy) [inline src/site/markdown/support.md:56]:
```suggestion
the Log4j products, individual committers may collaborate with services that provide such support.
```

File: src/site/markdown/support.md [hunk 2]
```diff
@@ -49,3 +49,16 @@ projects, we cannot recommend any particular individual for any specific issue.
...
+
+The following aims to be a list of all commercial support services involving one or more Log4j committers.
+
+### TideLift
```
Comment (vy) [inline src/site/markdown/support.md:60]:
```suggestion
### Tidelift
```

File: src/site/markdown/support.md [hunk 3]
```diff
@@ -49,3 +49,16 @@ projects, we cannot recommend any particular individual for any specific issue.
...
+
+Some Log4j maintainers are working with Tidelift to deliver commercial support and maintenance for the Open Source
+dependencies you use to build your applications. Learn more on the [TideLift website](https://tidelift.com).
```
Comment (vy) [inline src/site/markdown/support.md:63]:
```suggestion
Some Log4j maintainers receive funding from Tidelift for their maintenance efforts. See [the Tidelift website](https://tidelift.com) for details.
```

I don't think funded people deliver _commercial support_, hence I trimmed down the wording a bit.

**Comment by ppkarwasz** [comment]
Merged manually.

**Pull request closed without merging**